src/pybind/cephfs/cephfs.pyx

```diff
@@ -1150,6 +1150,37 @@ cdef class LibCephFS(object):
                           st_mtime=datetime.fromtimestamp(stx.stx_mtime.tv_sec),
                           st_ctime=datetime.fromtimestamp(stx.stx_ctime.tv_sec))
 
+    def lstat(self, path):
+        """
+        Get a file's extended statistics and attributes. When file's a
+        symbolic link, return the informaion of the link itself rather
+        than that of the file it points too.
+
+        :param path: the file or directory to get the statistics of.
+        """
+        self.require_state("mounted")
+        path = cstr(path, 'path')
+
+        cdef:
+            char* _path = path
+            statx stx
+
+        with nogil:
+            # FIXME: replace magic number with CEPH_STATX_BASIC_STATS
+            # 0x100 should make ceph_statx unfollow symlinks
```

Comment:
this magic number should be defined somewhere at the top of this file and reference the header file it comes from

Reply:
Done.